app/frontend/components/common/tooltip.tsx

```diff
@@ -10,3 +12,27 @@ const tooltip = (message, enabled, displayAlways?: boolean) => {
 }
 
 export default tooltip
+
+export const visitNufiTooltip = ({
+  content,
+  message,
+  style,
+}: {
+  content: string
+  message: string
+  style?: {[key: string]: string}
+}) => {
+  return (
+    <div className="nufi-tooltip-wrapper" style={style}>
+      <p>
+        <span {...tooltip(message, true)}>
+          <span className="show-info show-info-start">{''}</span>
+        </span>
+        {content}{' '}
+        <a href="http://adalite.io/nufi" target="_blank" rel="noopener">
+          Nufi
+        </a>
```

Comment:
Perhaps I'd extract this to a separate "link" component which would be imported to callers as well, so that the content is shown in one place. Example usage:
```
visitNufiTooltip({
            content: <>Manage accounts more easily with <NufiScreenLink /><>,
            ...
})
```